Makefile.packaging

```diff
@@ -8,6 +10,7 @@ GPG_PUBLIC_KEY      := .key
 DEB_DISTROS?=ubuntu-jammy-22.04 ubuntu-focal-20.04 ubuntu-bionic-18.04 debian-bullseye-11 debian-buster-10
 DEB_ARCHS?=arm64 amd64
 RPM_DISTROS?=centos-7-x86_64 redhatenterprise-7-x86_64 redhatenterprise-8-x86_64 redhatenterprise-9-x86_64 amazon-2-x86_64 amazon-latest-x86_64 amazon-2017.09-x86_64 suse-12-x86_64 suse-15-x86_64
```

Comment:
Can we remove the arch suffix at this point?

Reply:
I think the plan is to fix this in a future PR